datasync/chaindatafetcher/kafka/repository.go

```diff
@@ -39,7 +39,7 @@ type repository struct {
 func NewRepository(config *KafkaConfig) (*repository, error) {
 	kafka, err := NewKafka(config)
 	if err != nil {
-		logger.Error("Failed to create a new Kafka structure", "err", err, "brokers", config.Brokers, "partitions", config.Partitions, "replicas")
+		logger.Error("Failed to create a new Kafka structure", "err", err, "brokers", config.Brokers, "partitions", config.Partitions, "replicas", config.Replicas, "maxMessageBytes", config.SaramaConfig.Producer.MaxMessageBytes, "requiredAcks", config.SaramaConfig.Producer.RequiredAcks)
```

Comment:
Why don't you simply print `config.String()`?

Reply:
Added a string method. 9f4dd7b64c2024ad88c91a92a7e0a3665d7c231a
Thanks